tests/ParserTest.php

```diff
@@ -132,7 +133,8 @@ public static function parserErrorDataProvider(): iterable
 
         yield 'invalid content encoding' => [
             'expectedException' => new ParserException('Lexer exception "Feature file is not in UTF8 encoding" thrown for file /fake.feature'),
-            'featureText' => mb_convert_encoding('🔥 Все буде добре 🔥', 'EUC-JP', 'UTF-8'),
+            'featureText' => mb_convert_encoding('🔥 Все буде добре 🔥', 'EUC-JP', 'UTF-8')
+                ?: throw new RuntimeException('Could not encode string data for testing successfully'),
```

Comment:
I suggest reverting this. It will be solved by https://github.com/phpstan/phpstan-src/pull/4010 which makes the phpstan analysis smarter.